go.mod

```diff
@@ -10,6 +10,8 @@ require (
 	github.com/kr/pretty v0.1.0
 	github.com/kr/text v0.1.0 // indirect
 	github.com/vmware/vmw-guestinfo v0.0.0-20170707015358-25eff159a728
+	golang.org/x/tools v0.0.0-20201007032633-0806396f153e // indirect
```

Comment:
Doesn't look like the go.mod or go.sum changes are needed, can you revert those?

Reply:
Done. Looks like my IDE added some comfort features :-)